ehsm_kms_service/constant.js

```diff
@@ -11,6 +11,9 @@ const SM_SECRET_VERSION_STAGE_PREVIOUS = 0
 const SECRETNAME_LENGTH_MAX = 64
 const SECRETDATA_LENGTH_MAX = 4 * 1024
 const DESCRIPTION_LENGTH_MAX = 4 * 1024
+const QUOTE_LENGTH_MAX = 10924

```

Comment:
how do you get this value, you need to put some details here.

Reply:
the length after 8K base64,will be add desc